deploy/roles/r-core/meta/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+dependencies:
+  - {role: epel, when: "use_epel and not 'production' in group_names"}
+  - devtools
+  - shared
+  - common
```

Comment:
`common` can be removed here, since it is not a strict dependency of `r-core`, just `r-libs`